README_MACOS.md

```diff
@@ -44,7 +44,7 @@ Prerequisites:
   `xcode-select --install`
 - **homebrew** is recommended to install required libraries
   See http://brew.sh for installation instructions.
-- **git, cmake >= 3.5, libsndfile, readline, and qt5 >= 5.7**, installed via homebrew:
+- **git, cmake >= 3.12, libsndfile, readline, and qt5 >= 5.11**, installed via homebrew:
```

Comment:
How does this PR relate to required Qt version? The legacy build is built with 5.9.9 currently. I believe Qt 5.7 is still the minimum requirement in general (though not sure how feasible that is on macOS).

Reply:
i can't remember actually, will need to get back to you on this